changelog.md

```diff
@@ -14,6 +14,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Add `setId`, `createAttribute` and `getAttributesByType` inside `Component.js`.
   - Add `renameComponentId` inside `DefaultData.js`.
 
+- Add `arrangeComponentsPosition` method in `DefaultDrawer` and `DefaultPlugin`
```

Comment:
You miss to add `guides/migrations/0.18.0_to_0.X.0.md`
And show change in DefaultDrawer. You can read previous migration guides for example.

Reply:
What kind of change is expected from whoever uses DefautPlugin/DefaultDrawer ? I add a feature and do not modify anything that could be previously used.

Reply:
done